<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_ScriptHelp_Src</class>
 <widget class="QWidget" name="P_ScriptHelp_Src">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>240</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">
QToolButton{
	font: 75 10pt &quot;黑体&quot;;
	font-weight:bold;
	padding:2px;
	padding-left:6px;
	border: 1px solid transparent;
	background: rgba(255,255,255,0);
}
QToolButton:hover{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #fdecad, stop:0.3 #fde48c, stop:0.9 #fce58c, stop:1 #fdfadf )
}
QToolButton:pressed{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}
QToolButton:checked{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}
QToolButton:disabled{
	color:#bbbbbb;
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>插件没有资源路径设置。</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,1">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_src">
         <property name="styleSheet">
          <string notr="true">
/* == 群组控件 == */
QGroupBox{
	margin-top: 0.5em;
	padding-top: 20px;
	color:#d4449e;
	font-weight:bold;
    border: 1px solid #d4449e;
    border-radius: 4px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #FFFAFB, stop:1 #FEF0F3 );
}
QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left;
	top: 12px;
	left: 2px;
	margin-left: 0px;
	padding: 0 1px;
}
QGroupBox:disabled { border:1px solid #CCCCCC; background: #FFFFFF; }</string>
         </property>
         <property name="title">
          <string>资源路径</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_context">
         <property name="text">
          <string>没有描述。</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_sample">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#plainTextEdit_sample{
	background-color: transparent;
}</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../drillplugininformationreader.qrc"/>
 </resources>
 <connections/>
</ui>
